introduction/learning_resources.md

```diff
@@ -277,16 +278,26 @@ Courses | Duration | Effort
 
 ### Specializations
 
-After finishing the courses above, start your specializations on the topics that you have more interest.
+After finishing the courses above, start your specializations on the topics that you have more interest. Such as "Do I want to do Game development, software development, mobile app development, or white hat hacking?" 
 
 The following platforms currently offer specializations:
 
 #### edX: [xSeries](https://www.edx.org/xseries)
 #### Coursera: [Specializations](https://www.coursera.org/specializations)
 #### Udacity: [Nanodegree](https://www.udacity.com/nanodegree)
 #### FutureLearn: [Collections](https://www.futurelearn.com/courses/collections)
+#### Pluralsight: [Careers](https://www.pluralsight.com/browse/business-professional/careers)
 
-## More resource lists!
+## More resources and resource lists!
 Just like a good piece of software, this tutorial must be dependent on other lists. 
 
 - [Delevoper-Y videos](https://github.com/Developer-Y/cs-video-courses)
```

Comment:
I too am a software "Delevoper".

Reply:
Lol, okay, I'll fix it by tommorow night